people.rkt

```diff
@@ -585,6 +577,9 @@
               @(alumnus "Tony Garnock-Jones" #:year 2017
                         #:dissertation "http://syndicate-lang.org/tonyg-dissertation/"
                         #:personal-site "https://leastfixedpoint.com/")
+              @(alumnus "Kevin Clancy" #:year 2018
+                        #:extra "(Carnegie Mellon University)"
+                        #:personal-site "https://leastfixedpoint.com/")
```

Comment:
Personal site should be https://kevinclancy.github.io/

Reply:
fixed 1976b01